src/templates.ts

```diff
@@ -165,16 +165,14 @@ export const templates = (log: Logger, octokit: Pick<OctokitInstance, 'repos'>)
     log.debug('Extracting default configuration.')
 
     const defaults = await extract(loaded, 'defaults.yaml')
-    log.debug('Saw default configuration:')
-    log.debug(defaults)
+    log.debug(defaults, 'Saw default configuration.')
```

Comment:
```suggestion
    log.debug('Saw default configuration: %o', defaults)
```
Can we have such unification here?